lib/imgix/client.rb

```diff
@@ -59,7 +60,9 @@ def host_for_crc(path)
     end
 
     def host_for_cycle
-      @hosts_cycle = @hosts.cycle unless @hosts_cycle
```

Comment:
Incidentally, this line was throwing an initialization error that got in the way when we used `assert_output` to check for the expected deprecation warning.